source/trackers.json

```diff
@@ -1223,8 +1223,7 @@
         "mega.io": "meganz",
         "mega.nz": "meganz",
         "msidentity.com": "microsoft",
-        "e-msedge.net": "msedge",
-        "l-msedge.net": "msedge",
+        "msedge.net": "msedge",
```

Comment:
then the `s-msedge.net`, `e-msedge.net` and `l-msedge.net` domains will not be matched. But `msedge.net` can be added additionally

Reply:
I'm sorry for the confusion. `msedge.net` is already in the domains, but you can add `s-msedge.net`